slack-api-model/src/main/java/com/slack/api/util/json/GsonMessageAttachmentVideoHtmlFactory.java

```diff
@@ -2,13 +2,11 @@
 
 import com.google.gson.*;
 import com.slack.api.model.Attachment;
-import lombok.extern.slf4j.Slf4j;
 
 import java.lang.reflect.Type;
 import java.util.ArrayList;
 import java.util.List;
 
-@Slf4j
 public class GsonMessageAttachmentVideoHtmlFactory implements JsonDeserializer<Attachment.VideoHtml>, JsonSerializer<Attachment.VideoHtml> {
```

Comment:
Moved this class from slack-api-client project to slack-api-model project because the model project is more relevant for this class